.buildkite/bk.integration.pipeline.yml

```diff
@@ -225,6 +225,63 @@ steps:
         matrix:
           - default
 
+  - group: "Stateful:Debian"
+    key: integration-tests-debian
+    depends_on:
+      - integration-ess
+    steps:
+      - label: "x86_64:non-sudo: {{matrix}}"
+        depends_on: packaging-ubuntu-x86-64
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} false
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-debian-12"
+        matrix:
+          - default
+
+      - label: "x86_64:sudo: {{matrix}}"
+        depends_on:
+          - packaging-ubuntu-x86-64
+        # due to deb group present in matrix tar.gz and deb packages artifacts are required
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step packaging-ubuntu-x86-64
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-debian-12"
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+          - fleet-endpoint-security
+          - fleet-airgapped
+          - fleet-airgapped-privileged
+          - fleet-privileged
+          - fleet-upgrade-to-pr-build
+          - install-uninstall
+          # FQDN test failing at debian 12. Should be fixed in future.
+          #- fqdn
```

Comment:
If you disabled it, did you create the relevant GitHub issue in the `elastic-agent` repository? If so, I think the phrase "should be fixed in the future" should be replaced with a reference to the GitHub issue.

Additionally, let's follow this pattern: whenever something is disabled, we should create the relevant GitHub issue with as many details as possible. This includes the URL to the build failures, the stack trace, and any other information that could be helpful in case the URL for the build is no longer available in the future. This way, it will assist the person analyzing the failures.